Source/GitLab/gitlab-api.env

```diff
@@ -233,11 +233,11 @@ _t4dGitLabProjectAPIListBranches(){
 #       Write Merge Request Info to json file
 ###
 #   - Usage:
-#       * _t4dGitLabProjectAPIListBranches <User> <JsonFile> <supplemental query>
+#       * _t4dGitLabProjectAPIListBranches <Project> <JsonFile> <supplemental query>
 #
 ###
 #   - Example:
-#       * _t4dGitLabProjectAPIListBranches tk463 T4D-Sample-CMake toto.json
```

Comment:
did you tried to re-run doc generator to see if documentation is generated correctly ? using / in documentation might break stuffs

Reply:
Can't remember if I tried to gen the doc or not. Can you try it and tell me if anything goes bad ? My ubuntu always regens everything in a crappy order which makes it hard to check